app/uk/gov/hmrc/customs/declarations/stub/controllers/SecureTwoWayMessagingController.scala

```diff
@@ -451,21 +447,21 @@ object SecureTwoWayMessagingController {
         |         </div>
         |      </div>
         |   </div>
-        |   <h1 class="govuk-heading-l margin-top-small margin-bottom-small">about import doc</h1>
+        |   <h1 class="govuk-heading-l margin-top-small margin-bottom-small">MRN123456789098765 needs some action</h1>
```

Comment:
This seems an English message, but Welsh times

Reply:
This is user generated content so can not be translated